### PR TITLE
Update README.md for v1.0.0

### PR DESCRIPTION
**Pull request by bo3z**
# Description

> :memo: Fixes a minor mistake in the README.md. We can keep this PR open for any other changes required in the README before v.1.0.

## Type of change

For a new feature or function, please create an issue first to discuss it with us before submitting a pull request.

- [x] Documentation update

## Checklist

- [x] I have read the [guidelines for contributing](https://github.com/fastmachinelearning/hls4ml/blob/main/CONTRIBUTING.md).
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have installed and run `pre-commit` on the files I edited or added.
- [x] I have added tests that prove my fix is effective or that my feature works.

### CODE REVIEW

File: docs/setup.rst [hunk 1]
```diff
@@ -43,23 +43,30 @@ version can be installed directly from ``git``:
...
-* Xilinx Vivado HLS 2018.2 to 2020.1 for synthesis for Xilinx FPGAs
+* Xilinx Vivado HLS 2018.2 to 2020.1 for synthesis for Xilinx FPGAs using the ``Vivado`` backend.
+
+* Vitis HLS 2022.2 or newer is required for synthesis for Xilinx FPGAs using the ``Vitis`` backend.
 
-  * Vitis HLS 2022.2 or newer is required for synthesis for Xilinx FPGAs using the ``Vitis`` backend.
+* Intel Quartus 20.1 to 21.4 for the synthesis for Intel/Altera FPGAs using the ``Quartus`` backend.
 
-* Intel Quartus 20.1 to 21.4 for the synthesis for Intel FPGAs
+* oneAPI 2024.1 to 2025.0 with the FPGA compiler and recent Intel/Altara Quartus for Intel/Altera FPGAs using the ``oneAPI`` backend.
```
Comment (bo3z) [inline docs/setup.rst:67]: Typo: Altara

File: docs/api/auto.rst [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+It is initiated by configuring a precision in the configuration as 'auto'. Functions like :py:class:`~hls4ml.utils.config.config_from_keras_model` and :py:class:`~hls4ml.utils.config.config_from_onnx_model`
+automatically set most precisions to 'auto' if the ``'name'`` granularity is used.
+
+.. note::
+    It is recommended to pass the backend to the ``config_from_*`` functions so that they can properly extract all the configurable precisions.
+
+The approach taken by the precision inference is to set accumulator and other precisions to never truncate, using only the bitwidths of the inputs (not the values). This is quite conservative,
+especially in cases where post-training quantization is used, or if the bit widths were set fairly loosely. The recommended action in that case is to edit the configuration and explicitly set
+some widths in it, potentially in an iterative process after seeing what precisions are automatically set. Another option, currently implemented in :py:class:`~hls4ml.utils.config.config_from_keras_model`,
```
Comment (JanFSchulte) [inline docs/api/auto.rst:14]: I think this can updated now since we added it to QONNX and pytorch as well?

File: docs/api/configuration.rst [hunk 3]
```diff
@@ -45,8 +45,8 @@ This python dictionary can be edited as needed. A more advanced configuration ca
         default_precision='fixed<16,6>',
         backend='Vitis')
 
-This will include per-layer configuration based on the model. Including the backend is recommended because some configation options depend on the backend. Note, the precisions at the
-higher granularites usually default to 'auto', which means that ``hls4ml`` will try to set it automatically. Note that higher granularity settings take precendence
+This will include per-layer configuration based on the model. Including the backend is recommended because some configuration options depend on the backend. Note, the precisions at the
+higher granularites usually default to 'auto', which means that ``hls4ml`` will try to set it automatically (see :ref:`Automatic precision inference`). Note that higher granularity settings take precedence
 over model-level settings. See :py:class:`~hls4ml.utils.config.config_from_keras_model` for more information on the various options.
```
Comment (JanFSchulte) [inline docs/api/configuration.rst:50]: Should we generalize this part of the documentation to not just feature the keras parser as an example?

Comment (jmitrevs) [reply docs/api/configuration.rst]: I attempted to do this, though feel free to edit.

File: docs/api/auto.rst (hunk 2 above)
Comment (jmitrevs) [reply docs/api/auto.rst]: I updated this to not say that it's only for keras models.

File: docs/setup.rst (hunk 1 above)
Comment (jmitrevs) [reply docs/setup.rst]: Fixed.

File: README.md [hunk 4]
```diff
@@ -15,7 +15,9 @@ If you have any questions, comments, or ideas regarding hls4ml or just want to s
 
 # Documentation & Tutorial
 
-For more information visit the webpage: [https://fastmachinelearning.org/hls4ml/](https://fastmachinelearning.org/hls4ml/)
+For more information visit the webpage: [https://fastmachinelearning.org/hls4ml/](https://fastmachinelearning.org/hls4ml/).
+
+For introductory material on FPGAs, HLS and ML inferences using hls4ml, check out the [video](https://www.youtube.com/watch?v=2y3GNY4tf7A&ab_channel=SystemsGroupatETHZ%C3%BCrich).
```
Comment (jmitrevs) [inline README.md:20]: Does it make sense to also add this to the documentation, not just the readme?

**Review by JanFSchulte — APPROVED** [review]

**Pull request merged**